dotCMS/src/main/java/com/dotmarketing/util/ImportStarterUtil.java

```diff
@@ -0,0 +1,1148 @@
+                    if ("liferay.com".equals(c.getCompanyId())) {
+                        continue;
+                    }
+                    c.setKey(Base64.objectToString(Encryptor.generateKey()));
+
+                    try {
+                        c.setModified(true);
+                        CompanyManagerUtil.updateCompany(c);
+                    } catch (SystemException e) {
+                        throw new DotDataException("Unable to load company", e);
+                    }
+                }
+            } else if (_importClass.equals(Language.class)) {
+                for (Object aL : l) {
+                    final Language lang = (Language) aL;
+
+
+                    DotConnect dc = new DotConnect();
+                    dc.setSQL("insert into language (id,language_code,country_code,language,country) values (?,?,?,?,?)");
+                    dc.addParam(lang.getId());
+                    dc.addParam(lang.getLanguageCode());
+                    dc.addParam(lang.getCountryCode());
+                    dc.addParam(lang.getLanguage());
+                    dc.addParam(lang.getCountry());
+                    dc.loadResults();
+
+
+                }
+
+            } else if (_importClass.equals(UsersRoles.class)) {
+                for (Object role : l) {
+                    final UsersRoles userRole = (UsersRoles) role;
+
+                    // upsert pattern for the user role
+                    new DotConnect().setSQL("delete from users_cms_roles where user_id=? and role_id=?")
+                                    .addParam(userRole.getUserId())
+                                    .addParam(userRole.getRoleId())
+                                    .loadResult();
+                    new DotConnect().setSQL("insert into users_cms_roles (id,user_id,role_id) values (?,?,?)")
+                                    .addParam(userRole.getId())
+                                    .addParam(userRole.getUserId())
+                                    .addParam(userRole.getRoleId())
+                                    .loadResult();
+
+
+
+                }
+
+            }
+
+
+
+            else {
+                String id;
+                if (_importClass.equals(Relationship.class)) {
+                    id = "inode";
+                } else {
+                    _dh = new HibernateUtil(_importClass);
+                    id = HibernateUtil.getSession().getSessionFactory().getClassMetadata(_importClass)
+                                    .getIdentifierPropertyName();
+                    HibernateUtil.getSession().flush();
+                }
+
+                boolean identityOn = false;
+                String cName = _className.substring(_className.lastIndexOf(".") + 1);
+                String tableName = "";
+                if (classesWithIdentity.contains(cName) && DbConnectionFactory.isMsSql() && !cName.equalsIgnoreCase("inode")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons for EqualsIgnoreCase](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)